include/m_home.h

```diff
@@ -48,6 +49,12 @@ typedef struct Haniwa_c {
     /* 0x58 */ u32 bells; /* held bells from selling items */
 } Haniwa_c; // size = 0x5C
 
+typedef struct mHm_goki_c {
```

Comment:
Is this an original struct name?

Reply:
Yeah it's referenced on flashrom

Reply:
I wonder if we should start using a comment to signal this kind of stuff, something like this
```suggestion
// Original name is mHm_goki_c
typedef struct mHm_goki_c {
```

Reply:
We can decide later with the others, but I'll comment this one with that